sympy/printing/latex.py

```diff
@@ -2434,22 +2434,31 @@ def _print_Morphism(self, morphism):
         return "%s\\rightarrow %s" % (domain, codomain)
 
     def _print_TransferFunction(self, expr):
-        from sympy.core import Mul, Pow
-        num, den = expr.num, expr.den
-        res = Mul(num, Pow(den, -1, evaluate=False), evaluate=False)
-        return self._print_Mul(res)
+        num, den = self._print(expr.num), self._print(expr.den)
+        return r"\frac{%s}{%s}" % (num, den)
 
     def _print_Series(self, expr):
         args = list(expr.args)
         parens = lambda x: self.parenthesize(x, precedence_traditional(expr),
-                                             False)
+                                            False)
         return ' '.join(map(parens, args))
 
+    def _print_MIMOSeries(self, expr):
+        from sympy.physics.control.lti import MIMOParallel
+        args = list(expr.args)[::-1]
+        parens = lambda x: self.parenthesize(x, precedence_traditional(expr),
+                                             False) if isinstance(x, MIMOParallel) else self._print(x)
+        return r"\cdot".join(map(parens, args))
+
     def _print_Parallel(self, expr):
         args = list(expr.args)
-        parens = lambda x: self.parenthesize(x, precedence_traditional(expr),
-                                             False)
-        return ' '.join(map(parens, args))
+        func = lambda x: self._print(x)
+        return '+'.join(map(func, args))
+
+    def _print_MIMOParallel(self, expr):
+        args = list(expr.args)
+        func = lambda x: self._print(x)
+        return '+'.join(map(func, args))
```

Comment:
When you find yourself using `map` with `lambda` it is usually clearer just to use a comprehension:
```python
return '+'.join(self._print(arg) for arg in expr.args)
```
Sometimes `lambda` is useful for breaking long lines (I prefer not to have a multiline comprehension) but that doesn't apply here.

Should this have spaces around the `+` like `1 + 2`?